pom.xml

```diff
@@ -25,8 +25,8 @@
         <changelist>999999-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/opentelemetry-api-plugin</gitHubRepo>
         <opentelemetry.version>1.40.0</opentelemetry.version>
-        <opentelemetry-instrumentation.version>2.5.0</opentelemetry-instrumentation.version>
-        <opentelemetry-semconv.version>1.26.0-alpha</opentelemetry-semconv.version>
+        <opentelemetry-instrumentation.version>2.6.0</opentelemetry-instrumentation.version>
+        <opentelemetry-semconv.version>1.25.0-alpha</opentelemetry-semconv.version>
```

Comment:
We revert semconv to 1.25 as opentelemetry instrumentations have a dependency on semconv 1.25.